addons/sale_project/views/sale_order_views.xml

```diff
@@ -25,6 +25,7 @@
                     icon="fa-check-square-o"
                     invisible="not is_product_milestone or not project_ids or state == 'draft'"
                     groups="project.group_project_milestone"
+                    context="{'active_id': project_id}"
```

Comment:
```suggestion
                    context="{'default_project_id': project_id}"
```

Reply:
Default project is already set by the action, no need to add any. If the active key gets propagated, you can remove it directly in `action_view_milestone`.